iot2000/sw-description.bbin: update description file to support encry…

fahadarslan opened the pull request:
…ption

Signed-off-by: Fahad Arslan <Fahad_Arslan@mentor.com>

fahadarslan committed:
iot2000/sw-description.bbin: update description file to support encryption

Signed-off-by: Fahad Arslan <Fahad_Arslan@mentor.com>

chombourger commented on meta-iot2000-bsp/files/iot2000/sw-description.bbin:
```diff
@@ -8,21 +8,23 @@ software =
                                 sha256 = "@${image}";
                                 device = "/dev/mmcblk0p3";
-				compressed = true;
+                                compressed = true;
```
code in swupdate suggests that they do not support images that are both encrypted and compressed. did it work for you?

Noor-Ahsan replied:
He still need to test this on baord.

chombourger replied:
Does it mean that we no longer support non-encrypted SWUs?
Should we instead use a different bbin file when encryption is enabled?

fahadarslan replied:
Excerpt from proposed final bbin looks like:
```
+                                compressed = false;
+                                encrypted = @${image}-status;
```
So, if we have SWUPDATE_ENCRYPTION set to 1 in local.conf encrypted swu will be supported, otherwise un-encrypted swu will be supported.

fahadarslan committed:
wic-image.iot2000.wks: increase rootfs partition size

Increase rootfs partitions to make them 2G to allow updation of
large swu file.

Signed-off-by: Fahad Arslan <Fahad_Arslan@mentor.com>

fahadarslan committed:
iot2000/sw-description.bbin: keep compression off by default

SWUpdate currently has a limitation due to which an image that is
both encrypted and compressed is not supported. Since encryption
is one of the required features, disable compression in favour of
encryption.

Signed-off-by: Fahad Arslan <Fahad_Arslan@mentor.com>

fahadarslan committed:
iot2000/sw-description.bbin: remove unnecessary quotes

Signed-off-by: Fahad Arslan <Fahad_Arslan@mentor.com>

fahadarslan commented:
@chombourger, @Noor-Ahsan, kindly review the updated PR.

Noor-Ahsan commented:
Looks good
merging
